sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -36,64 +18,34 @@ Use the client library to:
 ### Prerequisites
 
 - An [Azure subscription][azure_sub].
-- An existing [Cognitive Services][cognitive_resource] or Text Analytics resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
-
-If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names:
-
-```PowerShell
-az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name> --sku <your-sku-name> --location <your-location>
-```
 
 ### Install the `@azure/ai-text-analytics` package
 
-Install the Azure Text Analytics client library for JavaScript with `npm`:
+Install the Azure Text Analytics Client library for Javascript with `npm`:
 
 ```bash
 npm install @azure/ai-text-analytics
 ```
 
 ### Create and authenticate a `TextAnalyticsClient`
 
-To create a client object to access the Text Analytics API, you will need the `endpoint` of your Text Analytics resource and a `credential`. The Text Analytics client can use either Azure Active Directory credentials or an API key credential to authenticate.
-
-You can find the endpoint for your text analytics resource either in the [Azure Portal][azure_portal] or by using the [Azure CLI][azure_cli] snippet below:
-
-```bash
-az cognitiveservices account show --name <your-resource-name> --resource-group <your-resource-group-name> --query "endpoint"
-```
-
-#### Using an API Key
-
-Use the [Azure Portal][azure_portal] to browse to your Text Analytics resource and retrieve an API key, or use the [Azure CLI][azure_cli] snippet below:
+To create a client object to access the Text Analytics API, you will need the `endpoint` of your Text Analytics resource and a `credential`. The Azure Text Analytics Client can use Azure Active Directory credentials to authenticate.
```

Comment:
How does the code generator know to talk about "endpoints" vs AAD vs "connection string". Same question for the "Client API key authentication is used in most of the examples," that comes a few lines later

Reply:
I think doing this, even if it is not true for all services, is better than nothing. I can remove it though.